rs-backend/src/date.rs

```diff
@@ -0,0 +1,39 @@
+use chrono::{UTC, DateTime};
+use serde::{self, Serialize, Deserialize};
+
+#[derive(Debug, Copy, Clone)]
+pub struct Date(pub DateTime<UTC>);
+
+// TODO: Deprecate and replace with RFC3339 dates.
```

Comment:
Leaving this as a separate change/for a future PR since it requires changing the tests and checking all the frontend pages; with possible changes to frontend code.